SeleniumPracticalExercise/PageObjects/EmployeeInformationPage.cs

```diff
@@ -28,36 +29,38 @@ public void AddButton()
         }
 
         /// <summary>
-        ///  txtbox Emp ID and insert stored ID
+        /// this is the second time on page where we...... 
+        /// search for emp id textbox and insert stored ID
+        /// then we click search button
         /// </summary>
         public void SetTxtBoxEmpID(string storeEmployeeID)
         {
             EditBoxSendKeysAndVerify(_employeeIDTxtBoxLocator, storeEmployeeID);
+            Click(_searchButtonLocator);
         }
 
-        public void SearchButton()
+        /// <summary>
+        ///  this is to verify ID # in search result
+        /// </summary>
+        public string verifyEmployeeId()
```

Comment:
This method name no longer matches what it's doing.

Reply:
forgot which method you were talking about here but I did go back and look at my method to see if they match what is it doing.